packages/components/src/Chips/InternalChipDismissible/hooks/useRepositionMenu.ts

```diff
@@ -0,0 +1,16 @@
+import { RefObject, useState } from "react";
+import { usePopper } from "react-popper";
+
+export function useRepositionMenu(attachTo: RefObject<Element | null>) {
+  const [positionElement, setPositionedElementRef] =
+    useState<HTMLElement | null>();
+  const popper = usePopper(attachTo.current, positionElement, {
+    modifiers: [
+      { name: "offset", options: { offset: [0, 8] } },
+      { name: "flip", options: { fallbackPlacements: ["top"] } },
+    ],
+    placement: "bottom",
```

Comment:
Was `placement: "bottom"` a default key value here?  Just read this here: https://popper.js.org/react-popper/v2/render-props/

Reply:
looks like it